styles/styles.css

```diff
@@ -20,7 +20,21 @@
   --text-color: black;
 
   /* fonts */
-  --body-font-family: "Avenir Next";
```

Comment:
are any of the changes in this file intended?

Reply:
I guess yes, I found some more general font sizes and families on the website, and I tought they would be helpful

Reply:
But why did you change this variable "back" to roboto?

Reply:
Oops, my mistake. Fixed it now